usecase/usecase-embedding-android-tests/tests.auto.xml

```diff
@@ -0,0 +1,78 @@
+        <description>
+          <bdd_test_script_entry test_script_expected_result="0">/opt/usecase-embedding-android-tests/testscripts/LoadAppFromManifestAsync.feature</bdd_test_script_entry>
+        </description>
+      </testcase>
+      <testcase component="Crosswalk Use Cases/Embedding API" execution_type="auto" id="OnCreateWindowRequested" platform="android" priority="P0" purpose="OnCreateWindowRequested Test With XWalkActivity" status="approved" type="functional_positive">
+        <description>
+          <bdd_test_script_entry test_script_expected_result="0">/opt/usecase-embedding-android-tests/testscripts/OnCreateWindowRequested.feature</bdd_test_script_entry>
+        </description>
+      </testcase>
+      <testcase component="Crosswalk Use Cases/Embedding API" execution_type="auto" id="OnCreateWindowRequestedAsync" platform="android" priority="P0" purpose="OnCreateWindowRequested Test With XWalkInitializer" status="approved" type="functional_positive">
+        <description>
+          <bdd_test_script_entry test_script_expected_result="0">/opt/usecase-embedding-android-tests/testscripts/OnCreateWindowRequestedAsync.feature</bdd_test_script_entry>
+        </description>
+      </testcase>
+      <testcase component="Crosswalk Use Cases/Embedding API" execution_type="auto" id="OnIconAvailableOnReceivedIcon" platform="android" priority="P0" purpose="onIconAvailable onReceivedIcon Test With XWalkActivity" status="approved" type="functional_positive">
+        <description>
+          <bdd_test_script_entry test_script_expected_result="0">/opt/usecase-embedding-android-tests/testscripts/onIconAvailableOnReceivedIcon.feature</bdd_test_script_entry>
+        </description>
+      </testcase>
+      <testcase component="Crosswalk Use Cases/Embedding API" execution_type="auto" id="OnIconAvailableOnReceivedIconAsync" platform="android" priority="P0" purpose="onIconAvailable onReceivedIcon Test With XWalkInitializer" status="approved" type="functional_positive">
+        <description>
+          <bdd_test_script_entry test_script_expected_result="0">/opt/usecase-embedding-android-tests/testscripts/onIconAvailableOnReceivedIconAsync.feature</bdd_test_script_entry>
+        </description>
+      </testcase>
+      <testcase component="Crosswalk Use Cases/Embedding API" execution_type="auto" id="Versions" platform="android" priority="P0" purpose="Versions Test With XWalkActivity" status="approved" type="functional_positive">
+        <description>
+          <bdd_test_script_entry test_script_expected_result="0">/opt/usecase-embedding-android-tests/testscripts/Versions.feature</bdd_test_script_entry>
+        </description>
+      </testcase>
+      <testcase component="Crosswalk Use Cases/Embedding API" execution_type="auto" id="VersionsAsync" platform="android" priority="P0" purpose="Versions Test With XWalkInitializer" status="approved" type="functional_positive">
```

Comment:
<status="approved" type="functional_positive" platform="android" priority="P0">
is no needed in tests.auto.xml, we should keep tests.auto.xml simplified.